packages/@ama-terasu/schematics/src/create-module/templates/package.json.template

```diff
@@ -22,7 +22,7 @@
     "@ama-terasu/core": "^<%= version %>"
   },
   "engines": {
-    "node": ">=14.0.0",
+     "node": ">=16.0.0",
```

Comment:
unfortunately there is an extra space added :D

Reply:
sad :(